lib/Core/ExecutorUtil.cpp

```diff
@@ -153,6 +153,8 @@ namespace klee {
     case Instruction::FCmp:
       assert(0 && "floating point ConstantExprs unsupported");
     }
+    
+    assert(0 && "unsupported expression in evalConstantExpr");
```

Comment:
This should probably be `llvm_unreachable("Unsupported expression in evalConstantExpr");`